v2/emailpassword/common-customizations/sessions/multiple-api-endpoints.mdx

```diff
@@ -28,9 +28,9 @@ To enable use of sessions for multiple API endpoints, you need to use the `sessi
 - `sessionTokenBackendDomain` in the frontend config must match the `cookieDomain` set in the backend config.
 :::
```

Comment:
not related to this page, but the docs for sharing sessions across sub domains also needs changing to say that even if you don't add a `.`, it will still share cookies across all frontend sub domains and their sub domains.